src/main/java/io_utilities/LogUtil.java

```diff
@@ -0,0 +1,29 @@
+package io_utilities;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/**
+ * The {@code LogUtil2} class provides utility methods for logging exceptions and stack traces using Log4j 2.
+ * It centralizes logging functionality to ensure consistent logging practices throughout the application.
+ */
+public class LogUtil {
```

Comment:
unnecessary class

In java the convention is to create a static logger object (`LogManager.getLogger`) inside each class and use it, so the class name is preserved in the class

The way you did makes the class name disappear in the logs